src/elm/Explorer/View.elm

```diff
@@ -1918,9 +1918,8 @@ viewEventUpdateEnqueuedDetails ctx event =
             row [ width fill ]
                 [ text <|
                     "Update the CCD to Euro conversion rate to "
-                        ++ String.fromInt microCCDPerEuro.denominator
-                        ++ " EUR = "
-                        ++ (T.amountToString <| T.amountFromInt microCCDPerEuro.numerator)
+                        ++ "1 EUR = "
+                        ++ (T.amountToString <| T.amountFromInt (microCCDPerEuro.numerator // microCCDPerEuro.denominator))
```

Comment:
Does this also address the weird rounding?

Because in this block https://dashboard.testnet.concordium.com/chain/6fbc2b29f5878a51834ff02dd9149267591655deb1e9a8e84120fa227c17b9ad the actual exchange rate is `1099511627776 / 14353114003361380859` which is not what is displayed on the network dashboard, so I assumed that either the JSON parsing already introduces loss of precision, or `amountFromInt` does.

Reply:
Hmm, it looks like it does fix it.
![Screenshot from 2022-02-17 09-48-30](https://user-images.githubusercontent.com/105212/154439143-75c7df5e-0d07-4bbe-af2a-1151029b91af.png)

I wonder where the rounding came from then.

Reply:
It must be amountFromInt that is essentially broken then for large values.